CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ cmake_minimum_required(VERSION 3.10.2)
 set(PROJECT_NAME libCellML)
 set(PROJECT_URL https://libcellml.org)
 set(_PROJECT_VERSION 0.2.0)
-set(PROJECT_DEVELOPER_VERSION -dev.19)
+set(PROJECT_DEVELOPER_VERSION -dev.21)
```

Comment:
why the version change in this PR?

Reply:
I think I forgot to merge the changes back in from the latest dev releases I made.  We need it at that level because that is what we have already used.   The release script is still a work in  progress.